wifi-subsys/components/uart/src/subsys_uart.c

```diff
@@ -31,6 +31,8 @@
 #include "at_handler.h"
 #include "wifi.h"
 #include "default_params.h"
+#include "serialization.h"
+#include "httpsclient.h"
```

Comment:
In the module or component file, if you are including others modules, you are breaking their independency, I refers to use uart_at, you will need to load component/tools and component/protocols